core/utils_test.py

```diff
@@ -691,15 +691,15 @@ def test_get_time_in_millisecs(self) -> None:
         self.assertEqual(
             dt,
             datetime.datetime.fromtimestamp(
-                python_utils.divide(msecs, 1000.0))) # type: ignore[no-untyped-call]
+                (msecs / 1000.0)))
```

Comment:
```suggestion
                msecs / 1000.0))
```
also this can be probably moved to previous line

ditto below